frontend/src/components/MainSearch/MainSearch.css

```diff
@@ -80,3 +80,43 @@
   font-weight: 500;
   margin: 0px;
 }
+
+.absoluteContainer {
+  /*display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100px;*/
+}
+
+.button-box{
+  margin-left: 88%;
+  cursor: pointer;
+  background-color: grey;
+  height: 30px;
+  width: 50px;
+}
+
+.chevron {
+  margin-top: -12px
+}
+
+.content{
```

Comment:
CSS is shared across all the components, so we generally want to use component-specific, descriptive names as opposed to super-generic ones like `content` and `button-box` and `close`.